instructure/canvas-android
# [Parent][MBL-13482] Add app secrets

This appears to be done the same exact way as we have been doing. The only secrets we really care about right now are the google-services.json, everything else is no longer being used. The docs for getting firebase (with the google-services.json) working say to put it in the platform specific modules (android, iOS) which allows us to keep using gradle with our buildSrc directory to merge in private data for the app. 

This also allows us to use all the dependencies in GlobalDependencies so that we can stay consistent for sdk targeting, generic android tools, and other things that should stay in sync between the apps.

To test:
There's not much to test here. Really just make sure that the flutter app can build. I did make some local changes to main.dart to verify that we are linked properly into firebase. You could as well by following some instructions here: https://firebase.google.com/docs/flutter/setup#try_out_an_example_app_with
